From c13b6e5bdec3b6883f2785e07fecc9bb7511a3c8 Mon Sep 17 00:00:00 2001
From: Federico Casabianca <federico.casabianca@gmail.com>
Date: Tue, 11 Feb 2014 18:16:34 +0100
Subject: [PATCH 2/2] First commit in the branch

---
 test05 | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 test05

diff --git a/test05 b/test05
new file mode 100644
index 0000000..c353be7
--- /dev/null
+++ b/test05
@@ -0,0 +1 @@
+New content for test01
-- 
1.8.3.4 (Apple Git-47)

